js/create_project_progress.jsx

```diff
@@ -0,0 +1,83 @@
+var DeployKeyTest = require('./deploy_key_test.jsx');
+
+var CreateProjectProgress = React.createClass({
+
+	checkInterval: false,
+
+	getInitialState: function() {
+		return {
+			complete: false,
+			checkingStatus: false
+		};
+	},
+
+	componentDidMount: function() {
+		this.checkInterval = window.setInterval(this.handleCheckStatus, 10000);
```

Comment:
It didn't clear because your CreateProjectProgress component did not unmount - it's still visible till the end. However, if you ever put that component under another component, and then the state changes in a way that removes it, the timer will keep ticking away in the background, and then fail. So you need it both in success (no need to poll after finishing) AND in unmount (as a best practice).